apps/shared/avifutil.h

```diff
@@ -82,6 +86,24 @@ avifAppFileFormat avifReadImage(const char * filename,
                                 uint32_t * outDepth,
                                 avifAppSourceTiming * sourceTiming,
                                 struct y4mFrameIterator ** frameIter);
+// Same as avifReadImage but the input file format is passed in instead of
+// being guessed from the file's name/content.
```

Comment:
Could we just add `inputFormat` arg to `avifReadImage` and have its current behavior if `inputFormat` is `AVIF_APP_FILE_FORMAT_UNKNOWN`? There would be no need for `avifReadFormat` then.